### PR TITLE
adds event for W16 teaparty and vol. meeting

### PR DESCRIPTION
**Pull request by fboxwala**

### CODE REVIEW

**Comment by fboxwala** [comment]
Im adding that the tea party is women-only when I get home. Don't merge till then, also see discussion on wics-events

**Comment by fboxwala** [comment]
@wics-uw/committers someone wanna review this? would be nice to deploy it before break

File: content/W2016/W16-event-teaparty.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Note: Although our volunteer meeting is open to everyone, our Tea Party is
+a Big CSters event is and is thus [women-only](http://wics.uwaterloo.ca/faq)
+
+## Event Details ##
+
++ **Who:** Everyone!
++ **What:** Tea Party and Volunteer Info
++ **Where:** CS Lounge (DC 3336)
++ **When:** Wed. Jan 6, 5:00&ndash;7:00PM
++ **Facebook Event Page:** Click [here](https://www.facebook.com/groups/wicsUW/).
```
Comment (ehashman) [inline content/W2016/W16-event-teaparty.md]: This isn't a FB event page. Save that for when we actually have one created?

File: content/W2016/W16-event-teaparty.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+title: Tea Party and Volunteer Meeting
+Date: 2016-06-01 17:00
```
Comment (ehashman) [inline content/W2016/W16-event-teaparty.md]: June 2016 sounds wrong to me for a posting date for this event, lol.

**Comment by fboxwala** [comment]
dates are hard.

File: content/W2016/W16-event-teaparty.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+Tags: social, volunteer, big csters
+Slug: tea-party-w16
+Author: Fatema Boxwala
+Summary: Join us for tea and for information on volunteering with us!
+
+Welcome back! You're cordially invited to our Tea Party and Volunteer meeting!
+Come out to socialize with some WiCS members, meet the new committee, and learn
+more about our volunteer opportunities! The volunteer meeting will run from
+5:00 - 5:30, and the Tea Party will go from 5:30 to 7:00. If you just want to
+come for the tea party, feel free to come by at 5:30! Please try to bring your
```
Comment (ehashman) [inline content/W2016/W16-event-teaparty.md]: Hating to nitpick further, but there are like 4 sentences ending in '!' in a row. Seems to be a bit overly-enthused :P

File: content/W2016/W16-event-teaparty.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+own mug if possible.
+
+Note: Although our volunteer meeting is open to everyone, our Tea Party is
+a Big CSters event is and is thus [women-only](http://wics.uwaterloo.ca/faq)
```
Comment (ehashman) [inline content/W2016/W16-event-teaparty.md]: Missed a . at the end of this line, whoops

**Comment by ehashman** [comment]
With additional things I missed from last time fixed, this is mergeable :<

**Comment by fboxwala** [comment]
fixed

**Comment by ehashman** [comment]
LGTM.

**Pull request merged**